File: video/rest/rooms/create-group-room-with-h264/create-group-room.7.x.java
import com.twilio.Twilio;
import com.twilio.rest.video.v1.Room;
import com.twilio.rest.video.v1.Room.RoomType;

public class Example {
    // Get your Account SID and Auth Token from https://twilio.com/console
    // To set up environment variables, see http://twil.io/secure
    public static final String API_KEY_SID = System.getenv("TWILIO_API_KEY_SID");;
    public static final String API_KEY_SECRET = System.getenv("TWILIO_API_KEY_SECRET");

    public static void main(String args[]) {
        // Initialize the client
        Twilio.init(API_KEY_SID, API_KEY_SECRET);

        Room room = Room.creator()
                .setUniqueName("DailyStandupWithH264Codec")
                .setType(RoomType.GROUP)
                .setVideoCodecs(Room.VideoCodec.forValue("H264"))
                .setStatusCallback("http://example.org")
                .create();

        System.out.println(room.getSid());
    }
}

File: video/rest/rooms/create-group-room/create-group-room.7.x.rb
# Download the Ruby helper library from twilio.com/docs/libraries/ruby
require 'twilio-ruby'

# Get your Account Sid and Auth Token from https://www.twilio.com/console
# To set up environmental variables, see http://twil.io/secure
api_key_sid = ENV['TWILIO_API_KEY']
api_key_secret = ENV['TWILIO_API_KEY_SECRET']

client = Twilio::REST::Client.new api_key_sid, api_key_secret

group_room = client.video.rooms.create(
  unique_name: 'DailyStandup',
  type: 'group',
  record_participants_on_connect: true,
  status_callback: 'http://example.org'
)

puts group_room
